props/saber_fett263_buttons.h

```diff
@@ -2073,6 +2076,18 @@ SaberFett263Buttons() : PropBase() {}
   }
 #endif
 
+  void DoCopyPreset() {
```

Comment:
"CopyPreset" and "DoCopyPreset" sounds like they do the same thing.
Add comments to explain what they do, and make sure that it clarifies the difference.

Reply:
consolidated to one function, only difference is whether it's used in menu or not, when used outside menu we need an "announce" when in the menu this is not used.

Reply:
That works, but even better would have been to do it like:

CopyPreset() { ... }
CopyPresetAndAnnounce() { CopyPreset();  /* do the announcement here */ }